lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -46,7 +46,7 @@ export default TestModule.extend({
       this.setupSteps.push(this.setupComponentUnitTest);
     } else {
       this.callbacks.subject = function() {
-        throw new Error("component integration tests do not support `subject()`.");
+        throw new Error("component integration tests do not support `subject()`. Instead, render the component as if it were HTML: `this.render('<my-component foo=true>');`. For more information, read: http://guides.emberjs.com/v2.2.0/testing/testing-components/");
```

Comment:
@rwjblue replaced the link with http://guides.emberjs.com/v2.2.0/testing/testing-components/

Also, http://guides.emberjs.com/v2.2.0/ looks great!